metron-interface/metron-alerts/src/app/alerts/alerts-list/table-view/table-view.component.scss

```diff
@@ -38,3 +38,20 @@
   padding: 0.3rem 3px;
 }
 
+.below-table-area {
+  display: flex;
+  justify-content: center;
+}
+
+.no-result-box {
+  position: absolute;
```

Comment:
Instead of using all this absolute positioning stuff with transforms, couldn't you do something like the following?
```
    align-items: center;
    display: flex;
    flex-direction: column;
    padding-top: 50px;
    width: 200px;
    
```

Might be a little more reliable in case of browser resizing.